roles/chair-and-vice.md

```diff
@@ -85,4 +85,4 @@ When creating a new event or updating a old one.
    For discussion sessions those are all the instructors that you invited.
 
 [repository]: https://github.com/swcarpentry/board/
-[minutes-repository]: https://github.com/swcarpentry/board/tree/master/subcommittee/mentoring/
+[minutes-repository]: https://github.com/carpentries/mentoring/minutes
```

Comment:
I think you need `tree/master` so this should be: https://github.com/carpentries/mentoring/tree/master/minutes

Reply:
fixed in eb0c57b261b37b028fbbf15aa103f3794a68704d